sdk/samples/iot/docs/how_to_iot_hub_samples_linux.md

```diff
@@ -44,51 +46,53 @@ To run the samples, ensure you have the following programs and tools installed o
 
     ```bash
     sudo apt-get update
-    sudo apt-get install build-essential # make and gcc
-    sudo apt-get install curl unzip tar pkg-config
-    sudo apt-get install git
-    sudo apt-get install openssl libssl-dev
+    sudo apt-get install build-essential curl unzip tar pkg-config git openssl libssl-dev
     ```
 
 ## Setup Instructions
 
-1. Install Microsoft [vcpkg](https://github.com/microsoft/vcpkg) package manager and [Eclipse Paho MQTT C client](https://www.eclipse.org/paho/). This installation may take an extended amount of time (~20-30 minutes).
+1. Install Microsoft [vcpkg](https://github.com/microsoft/vcpkg) package manager and [Eclipse Paho MQTT C client](https://www.eclipse.org/paho/). This installation may take an extended amount of time (~15-20 minutes).
 
     ```bash
-    cd / # Run this command from any directory to go to the root.
-    /$ sudo git clone https://github.com/Microsoft/vcpkg.git
-    /$ cd vcpkg
-    /vcpkg$ sudo ./bootstrap-vcpkg.sh
-    /vcpkg$ sudo ./vcpkg install --triplet x64-linux curl cmocka paho-mqtt
-    /vcpkg$ cd ..
+    cd ~ # Run this command from any directory to go to your user home directory.
+    ~$ sudo git clone https://github.com/Microsoft/vcpkg.git
+    ~$ cd vcpkg
+    ~/vcpkg$ sudo ./bootstrap-vcpkg.sh
+    ~/vcpkg$ sudo ./vcpkg install --triplet x64-linux curl cmocka paho-mqtt
+    ~/vcpkg$ cd ..
     ```
 
 2. Set the vcpkg environment variables.
 
     ```bash
-    /$ export VCPKG_DEFAULT_TRIPLET=x64-linux
-    /$ export VCPKG_ROOT=/vcpkg
+    ~$ export VCPKG_DEFAULT_TRIPLET=x64-linux
+    ~$ export VCPKG_ROOT=~/vcpkg
     ```
 
     NOTE: Please keep in mind, **every time a new terminal is opened, the environment variables will have to be reset**.
 
-3. Install the latest version of [CMake](https://cmake.org/download).  In this sample, it is `cmake-3.18.2.tar.gz`. This installation may also take an extended amount of time (~20-30 minutes).
+3. Install [CMake](https://cmake.org/files).
```

Comment:
Link to the download page instead:
```suggestion
3. Install [CMake](https://cmake.org/download/).
```

Reply:
I went with files in this case because I wanted the user to see the options they will be using wget on in the first command.  
```
wget https://cmake.org/files/v3.18/cmake-3.18.3-Linux-x86_64.sh
```

For Windows I do keep it with download since that has the Windows installer easily accessible.